src/R/Components/Impl/Plots/Implementation/Commands/PlotDeviceCommand.cs

```diff
@@ -19,7 +19,7 @@ internal abstract class PlotDeviceCommand : InteractiveWorkflowAsyncCommand {
 
         protected bool IsInLocatorMode {
             get {
-                return VisualComponent.LocatorMode;
+                return VisualComponent.Device.LocatorMode;
```

Comment:
Thanks. Yes it can. I didn't notice because VS command handling swallows that exception.